packages/reanalyze/reanalyze.2.23.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "Dead values/types, exception, and termination analysis for OCaml/ReScript"
+description: """
+Experimental analyses for ReScript/OCaml/Reason: for globally dead values/types, exception analysis, and termination analysis.
+"""
+maintainer: ["Cristiano Calcagno"]
+authors: ["Cristiano Calcagno"]
+license: "MIT"
+homepage: "https://github.com/rescript-association/reanalyze"
+bug-reports: "https://github.com/rescript-association/reanalyze/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.08" & < "4.15"}
+  "cppo" {build}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/rescript-association/reanalyze.git"
```

Comment:
```suggestion
conflicts: [
  "base-domains"
  "base-effects"
]
dev-repo: "git+https://github.com/rescript-association/reanalyze.git"
```

For 
- domains: [reanalyze.2.23.0 (failed: The constructor Kept expects 2 argument(s),)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/8b788b03c6ef4115193a3b022d1d96fec7aa78ed/variant/opam-2.1,extras,domains,reanalyze.2.23.0)
- domains-effects: [reanalyze.2.23.0 (failed: The constructor Typedtree.Texp_match expects 4 argument(s),)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/8b788b03c6ef4115193a3b022d1d96fec7aa78ed/variant/opam-2.1,extras,domains-effects,reanalyze.2.23.0)